recipes/gperftools/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "gperftools" %}
+{% set version = "2.7" %}
+{% set sha256 = "3a88b4544315d550c87db5c96775496243fb91aa2cea88d2b845f65823f3d38a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/{{name}}/{{name}}/archive/{{name}}-{{version}}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - gcc 4.8.5
+    - libgcc 4.8.5
```

Comment:
You probably don't need `gcc` and definitely don't `libgcc` here :smile: (`libgcc` is a runtime dep only.)